App/Containers/Wallet.js

```diff
@@ -19,16 +19,17 @@ class Wallet extends React.PureComponent {
     const params = navigation.state.params || {}
     const greeting = params.username ? 'Hello, ' + params.username : 'Hi there!'
     const headerTitle = (
-      <Text style={navStyles.headerTitle}>
-        {greeting}
-      </Text>
+      <View style={navStyles.headerCenter}>
```

Comment:
There is a property on the navigationOptions called `headerTitleStyle` where you can pass in styles, including margin, padding, etc, and avoid having to do this view wrapper everywhere.

https://reactnavigation.org/docs/en/headers.html#adjusting-header-styles

Reply:
i assumed something was getting lost here, because we are setting that property in the navigator creation step

Reply:
I don't see that. I only see this:
```
headerTitle: {
    fontFamily: 'BentonSans-Bold',
    fontSize: 17,
    paddingTop: 8, // <- needed because BentonSans has that weird bottom gap
    color: Colors.charcoal
  }
```

Reply:
Nothing about margin or horizontal anything

Reply:
That is from `NavigationStyles.js`

Reply:
so it was never working? interesting